### PR TITLE
Add SignalR Functions extension contact

### PR DESCRIPTION
**Pull request by Y-Sindo**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -575,6 +575,10 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 # ServiceLabel: %SignalR %Service Attention
 /sdk/signalr/Microsoft.Azure.Management.SignalR/            @sffamily @chenkennt
 
+# ServiceLabel: %SignalR %Service Attention
```
Comment (jsquire) [inline .github/CODEOWNERS:578]:
```suggestion
```

Comment (jsquire) [reply .github/CODEOWNERS:578]: There can't be two annotations for the same "Service Attention" + label combination.  The bot is unable to consider paths; it just keys on the label combination.

Comment (Y-Sindo) [reply .github/CODEOWNERS:578]: I'd like to make issues like #27414 to be routed to me, but only path under "/sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/".  Can PR label do that?

Comment (jsquire) [reply .github/CODEOWNERS:578]: Unfortunately, it isn't possible to filter service attention by path.  The closest that you'd be able to achieve is to be part of the mention list for anything tagged `SignalR` and `Service Attention`.

**Review by jsquire — APPROVED** [review]

**Pull request merged**